src/ProjectOrigin.WalletSystem.Server/Models/CertificatesFilter.cs

```diff
@@ -1,9 +1,11 @@
 using System;
+using ProjectOrigin.WalletSystem.Server.Services.REST.v1;
 
 namespace ProjectOrigin.WalletSystem.Server.Models;
 
-public record CertificatesFilter
+public record CertificatesFilter(SliceState State)
```

Comment:
Why is this property written as a positional property?

Reply:
Because you have to set it